Automattic/jetpack
# [Subscription block] Fix PHP Warning

Fixes #27882

#### Changes proposed in this Pull Request:
This PR fixes a PHP Warning that's caused when using the old subscription shortcode, and no `include_social_followers` parameter is set.

#### Other information:


#### Jetpack product discussion
https://github.com/Automattic/jetpack/issues/27882

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

1. Make sure your site is on the latest Jetpack release (or trunk)
2. On your site, create a new post and add a Shortcode block with following contents:

```
[jetpack_subscription_form show_subscribers_total=true]
```

3. Navigate to this post on the frontend
4. Examine the logs (using the CLI)
5. You'll see

```
[12-Dec-2022 11:54:37 UTC] PHP Warning:  Undefined array key "include_social_followers" in /srv/htdocs/wp-content/plugins/jetpack/modules/subscriptions/views.php on line 860
```

6. Checkout this branch, build the Jetpack plugin and (if using Atomic) sync it to your website
7. Refresh the browser
8. Verify that you didn't get a new warning
